frontend/src/components/accountPage/UserList/UserList.tsx

```diff
@@ -0,0 +1,304 @@
+          onFilter={makeFilter('name')}
+          filterDropdown={({
+            setSelectedKeys,
+            selectedKeys,
+            confirm,
+            clearFilters,
+          }) => (
+            <div style={{ padding: 8 }}>
+              <Input
+                placeholder="Search by Name"
+                onChange={e => handleOnChange(e, setSelectedKeys)}
+                value={selectedKeys[0]}
+                onPressEnter={handleOnPressEnter(
+                  confirm,
+                  selectedKeys,
+                  setSearchText
+                )}
+              />
+              <Space>
+                <ResetButton
+                  clearFilters={clearFilters}
+                  setSearchText={setSearchText}
+                >
+                  Reset
+                </ResetButton>
+              </Space>
+            </div>
+          )}
+          width={120}
+        />
+        <Column
+          title="Surname"
+          dataIndex="surname"
+          sorter={(a: User, b: User) => a.surname.localeCompare(b.surname)}
+          key="surname"
+          filterIcon={<SearchOutlined />}
+          onFilter={makeFilter('surname')}
+          filterDropdown={({
+            setSelectedKeys,
+            selectedKeys,
+            confirm,
+            clearFilters,
+          }) => (
+            <div style={{ padding: 8 }}>
+              <Input
+                placeholder="Search by Surname"
+                onChange={e => handleOnChange(e, setSelectedKeys)}
+                value={selectedKeys[0]}
+                onPressEnter={handleOnPressEnter(
+                  confirm,
+                  selectedKeys,
+                  setSearchText
+                )}
+              />
+              <Space>
+                <ResetButton
+                  clearFilters={clearFilters}
+                  setSearchText={setSearchText}
+                >
+                  Reset
+                </ResetButton>
+              </Space>
+            </div>
+          )}
+          width={120}
+        />
+        <Column
+          title="Email"
+          dataIndex="email"
+          filterIcon={<SearchOutlined />}
+          onFilter={makeFilter('email')}
+          filterDropdown={({
+            setSelectedKeys,
+            selectedKeys,
+            confirm,
+            clearFilters,
+          }) => (
+            <div style={{ padding: 8 }}>
+              <Input
+                placeholder="Search by Email"
+                onChange={e => handleOnChange(e, setSelectedKeys)}
+                value={selectedKeys[0]}
+                onPressEnter={handleOnPressEnter(
+                  confirm,
+                  selectedKeys,
+                  setSearchText
+                )}
+              />
+              <Space>
+                <ResetButton
+                  clearFilters={clearFilters}
+                  setSearchText={setSearchText}
+                >
+                  Reset
```

Comment:
Remove this text (check other occurrences) since it is already included in the button itself